keccak256/src/gates/iota_b13.rs

```diff
@@ -0,0 +1,220 @@
+use halo2::plonk::Instance;
+use halo2::{
+    circuit::Region,
+    plonk::{Advice, Column, ConstraintSystem, Error, Selector},
+    poly::Rotation,
+};
+use pasta_curves::arithmetic::FieldExt;
+use std::marker::PhantomData;
+
+#[derive(Clone, Debug)]
+pub struct IotaB13Config<F> {
+    q_enable: Selector,
+    state: [Column<Advice>; 25],
+    round_ctant_b13: Column<Advice>,
+    round_constants: Column<Instance>,
+    _marker: PhantomData<F>,
+}
+
+impl<F: FieldExt> IotaB13Config<F> {
+    // We assume state is recieved in base-9.
+    pub fn configure(
+        q_enable: Selector,
+        meta: &mut ConstraintSystem<F>,
+        state: [Column<Advice>; 25],
+        round_ctant_b13: Column<Advice>,
+        round_constants: Column<Instance>,
+    ) -> IotaB13Config<F> {
+        meta.create_gate("iota_b13", |meta| {
+            // def iota_b13(state: List[List[int], round_constant_base13: int):
+            // state[0][0] += round_constant_base13
+            // return state
+            let state_00 = meta.query_advice(state[0], Rotation::cur())
+                + meta.query_advice(round_ctant_b13, Rotation::cur());
```

Comment:
Can we assign the `round_ctant_b13` as constants here?
```suggestion
                + Expression::Constant(F::from(round_ctant_b13));
```